R/Lithuania.R

```diff
@@ -0,0 +1,407 @@
+#'  increases by about 20-40% in a few days. Such a "delay" in the data is
+#'  natural: for example, for those who died last night, a death certificate
+#'  is likely to be issued as soon as this report is published this morning.
+#'
+#' @section De jure and statistical counts:
+#'
+#' Beginning in February 2021 the OSP makes statistical estimates
+#' of the number of recovered and active cases, since review of the data
+#' showed that some cases individuals still considered as active cases
+#' had recovered, but not documented or registered as such.
+#'
+#' These are listed as by the OSP as `active_de_jure` and
+#' `recovered_de_jure` (officially still considered sick),
+#' and `active_sttstcl` and `recovered_sttstcl` (an estimate of how
+#' many of these are still ill).
+#'
+# nolint start
+#' @source https://opendata.arcgis.com/datasets/d49a63c934be4f65a93b6273785a8449_0
+# nolint end
+#' @examples
+#' \dontrun{
+#' region <- Lithuania$new(verbose = TRUE, steps = TRUE, get = TRUE)
+#' }
+#' @export
+Lithuania <- R6::R6Class("Lithuania",
+  inherit = DataClass,
+  public = list(
+
+    # Core Attributes
+    #' @field country name of country to fetch data for
+    country = "Lithuania",
+    #' @field supported_levels A list of supported levels.
+    supported_levels = list("1", "2"),
+    #' @field supported_region_names A list of region names in order of level.
+    supported_region_names = list(
+      "1" = "county", # apskritis
+      "2" = "municipality" # savivaldybe
+    ),
+    #' @field supported_region_codes A list of region codes in order of level.
+    supported_region_codes = list(
+      "1" = "iso_3166_2", # default is to just give county data
+      "2" = "iso_3166_2_municipality" # if giving municipality data
+    ),
+    #' @field common_data_urls List of named links to raw data that are common
+    #' across levels.
+    common_data_urls = list(
+      "main" = "https://opendata.arcgis.com/datasets/d49a63c934be4f65a93b6273785a8449_0.csv" # nolint
+    ),
+    #' @field source_data_cols existing columns within the raw data
+    source_data_cols = c(
+      "cases_new", "tested_new", "recovered_total", "deaths_new"
+    ),
+    # Additional attributes specific to the Lithuania functionality
+    #' @field death_definition which criteria of deaths attributed to
+    #' COVID to use
+    death_definition = "of",
+    #' @field recovered_definition whether to use the official counts of
+    #' recovered cases or the statistical estimates provided by OSP
+    recovered_definition = "official",
+    #' @field all_osp_fields whether to return all the data vectors provided
+    #' by OSP
+    all_osp_fields = FALSE,
+    #' @field national_data whether to return data rows for national results
+    national_data = FALSE,
+
+    #' @description Set up a table of region codes for clean data
+    set_region_codes = function() {
+      self$codes_lookup <- list(
+        "1" = lithuania_codes,
```

Comment:
As this is exactly the same maybe we don't need list("1", "2") and can just directly assign to codes_lookup. I know it breaks the package structure a very little but seems forgivable....